e2e/scenario/validators.go

```diff
@@ -47,6 +48,34 @@ func SysctlConfigValidator(customSysctls map[string]string) *LiveVMValidator {
 	}
 }
 
+func KubeletConfigValidator(kubeletConfigParams map[string]string) *LiveVMValidator {
+	keysToCheck := make([]string, 0)
+	for k, _ := range kubeletConfigParams {
+		keysToCheck = append(keysToCheck, k)
+	}
+	return &LiveVMValidator{
+		Description: "assert kubelet config settings",
+		Command:     fmt.Sprintf("jq `{%s}` /etc/default/kubeleconfig.json", strings.Join(keysToCheck, ",")),
```

Comment:
what are you trying to achieve here? just spit out the file? `jq . /etc/default/kubeletconfig.json` or simply `cat /etc/default/kubeletconfig.json` ? 

backticks embedded inside double quotes will result in the contents of the backticks being executed as a command.

the brackets seem pointless/purely syntax error

the keys in the map do not match the keys in the top level of the file

even if they did, you do not prepend them with a dot for proper field access.

```bash
root@Ubuntu-2204-jammy-amd64-base ~ # echo '{"foo": "bar", "memory-manager-policy": "foo"  }' | jq `memory-manager-policy`
{
  "foo": "bar",
  "memory-manager-policy": "foo"
}
memory-manager-policy: command not found
root@Ubuntu-2204-jammy-amd64-base ~ # echo '{"foo": "bar", "memory-manager-policy": "foo"  }' | jq `{memory-manager-policy}`
{memory-manager-policy}: command not found
{
  "foo": "bar",
  "memory-manager-policy": "foo"
}
root@Ubuntu-2204-jammy-amd64-base ~ # echo '{"foo": "bar", "memory-manager-policy": "foo"  }' | jq '{memory-manager-policy}'
jq: error: syntax error, unexpected '-', expecting '}' (Unix shell quoting issues?) at <top-level>, line 1:
{memory-manager-policy}
jq: 1 compile error
```

Reply:
The goal was to print out keys that are added in custom config from kubelet config json that's on the node. When node-shelled into the node, it worked as expected. But when the command is triggered during test there is an error. 

Then that command outputs a json in stdout that I can unmarshal to check values